src/scripts/config.js

```diff
@@ -1,6 +1,9 @@
 (function () {
   'use strict';
 
+  var coachBasePath = '//openstax.github.io/concept-coach/';
```

Comment:
I believe CNX has a policy of not depending on other sites (except google analytics and I guess now zendesk) but others might be able to chime in.